script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -273,7 +273,7 @@ It seems that this person is no different from a cat that does not know left fro
 "As you please, Akiha-sama."
 <Page79>
 After bowing to Akiha, Hisui comes towards me without any hesitation.
-Her line of sight is now directed at me, so I look all over the place so that our eyes don't meet.
+Her line of sight is now directed at me, and I look all over the place so that our eyes don't meet.
```

Comment:
How do you read into that phrase then? 
> Original:
> 視線は俺に向けられているが、こちらと視線が合わないように全体を見据えている。

> 全体 (ぜんたい) (n-adv,n-t) whole; entirety; whatever (is the matter); (P); 
> 見据える (みすえる) (v1,vt) (1) to stare fixedly at; to fix one's gaze on; (2) to make sure of; to set one's eyes on (e.g. the future); to focus on; 

"Looking all over the place so that our eyes don't meet" seems like a reasonable conversion of a raw "Our gaze / doesn't meet / in order to, whole / gaze at" unless zentai here refers to all of Akiha for some reason?